themes/default/content/docs/concepts/resources/names.md

```diff
@@ -323,7 +323,7 @@ error: Duplicate resource URN 'urn:pulumi:production::acmecorp-website::custom:r
 
 Any change to the URN of a resource causes the old and new resources to be treated as unrelated—the new one will be created (since it was not in the prior state) and the old one will be deleted (since it is not in the new desired state). This behavior happens when you change the `name` used to construct the resource or the structure of a resource’s parent hierarchy.
 
-Both of these operations will lead to a different URN, and thus require the `create` and `delete` operations instead of an `update` or `replace` operation that you would use for an existing resource. In other words, be careful when you change a resource’s name.
+Both of these operations will lead to a different URN, and thus require the `create` and `delete` operations instead of an `update` or `replace` operation that you would use for an existing resource. In other words, be careful when you change a resource’s name because **changing the name of a resource will create a brand new resource and delete the old/original resource**.
```

Comment:
At least as of today, we rarely use bold for emphasis, so I'd probably recommend sticking to that policy for now, just for consistency:

```suggestion
Both of these operations will lead to a different URN, and thus require the `create` and `delete` operations instead of an `update` or `replace` operation that you would use for an existing resource. In other words, be careful when you change a resource’s name because changing the name of a resource will create a new resource and delete the old/original resource.
```

That said, this does seem like something worth calling out -- perhaps even as an alert, since I believe the user who filed it was surprised by this behavior. We have a `{{< notes >}}` shortcode that we use for callouts that takes an optonal `type` param (`info` or `warning`) that might be a good fit for this. What do you think? Here's an example:

https://www.pulumi.com/docs/concepts/stack/#generate-an-update-plan

![image](https://github.com/pulumi/pulumi-hugo/assets/274700/92c36a2b-40c7-41d8-85fe-6275bf4d01e5)

It may also be worth mentioning that by default, we create new resources (and update existing resources' relationships to them) before deleting old ones. (Deleting resources is a bit less scary when you know this.)

Reply:
I really like the idea of putting that extra warning in a "warning"-type note callout.